src/containers/pages/FocusInvestigation/active/tests/index.test.tsx

```diff
@@ -25,6 +26,7 @@ jest.mock('../../../../../configs/env');
 describe('containers/pages/ActiveFocusInvestigation', () => {
   beforeEach(() => {
     jest.resetAllMocks();
+    MockDate.reset();
```

Comment:
Need to reset mocks that `new Date()` returns the current datetime for this test suite